core/src/templates/components/card/card.json

```diff
@@ -4,7 +4,7 @@
   "card_headline": "Headline goes here",
   "card_body": "<p>Sample text goes here and this card should be able to handle a few lines.</p>",
   "card_link": "https://stanford.edu",
-  "card_cta_attributes": " rel='no-follow'",
+  "card_cta_attributes": "rel='no-follow'",
```

Comment:
In >90% of our templates, we put a space in the `attributes`' value, and put no space between the tag and `{{ attributes }}` in the `.twig` file. I think we should stay consistent.

Reply:
Hmmmm....is there a reason why we do that for attributes? The no space thing actually screwed up things for me quite a few times especially when the component is nested. Perhaps we should always leave a space instead?

Reply:
Having no space between the html element and the `{{ attributes }}` variable in the twig file is a Drupal standard that doesn't pair well with his system or others. I am happy to add the space to the twig templates and remove the space from our data. Seems cleaner that way anyhow.

Reply:
I see - didn't realize that's a Drupal thing, @sherakama 😄  Yeah, if it's not too much work for you guys, it would be nice to add the space to the attribute so it's consistent with the other variables. Thanks!

Reply:
Yeah, that'd be great if we could put the space in the template vs. the data.